app-information.html

```diff
@@ -0,0 +1,369 @@
+        </p>
+      </section>
+      <section>
+        <h3>
+          <code title="">description</code> member
+        </h3>
+        <p>
+          The <dfn>description</dfn> member allows the developer to describe
+          the purpose of the web application.
+        </p>
+      </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
+        </h3>
+        <p>
+          The <dfn>iarc_rating_id</dfn> member is a <a>string</a> that
+          represents the <a href="https://www.globalratings.com/">International
+          Age Rating Coalition (IARC)</a> certification code of the web
+          application. It is intended to be used to determine which ages the
+          web application is appropriate for.
+        </p>
+        <p class="note">
+          An IARC certificate can be obtained via participating storefronts,
+          intended to be used for distributing the web app. The
+          <a>iarc_rating_id</a> member only takes a single certification code.
+          The same code can be shared across participating storefronts, as long
+          as the distributed product remains the same (i.e., doesn’t serve
+          totally different code paths depending on user agent sniffing and the
+          like) and the other storefronts support it.
+        </p>
+        <div class="informative">
+          <p>
+            The following shows a very simple <a>manifest</a> with the
+            <code>iarc_rating_id</code> member.
+          </p>
+          <pre class="example json" title="very simple manifest">
+          {
+            "name": "Donate App",
+            "description": "This app helps you donate to worthy causes.",
+            "iarc_rating_id": "e84b072d-71b3-4d3e-86ae-31a8ce4e53b7",
+            "icons": [{
+              "src": "images/icon.png",
+              "sizes": "192x192"
+            }]
+          }
+        </pre>
+        </div>
+        <p>
+          More information on the IARC can be found at: <a href=
+          "https://www.globalratings.com/how-iarc-works.aspx">How IARC
+          Works</a> and <a href=
+          "https://www.globalratings.com/for-developers.aspx">How developers
+          can get their games and apps rated with IARC</a>.
+        </p>
+      </section>
+      <section>
+        <h3>
+          <code title="">screenshots</code> member
+        </h3>
+        <p>
+          The <dfn>screenshots</dfn> member is an <a>array</a> of
+          {{ManifestImageResource}}s, representing the web application in
+          common usage scenarios.
+        </p>
+        <p>
+          For accessibility purposes, authors MUST provide a
+          {{ImageResource/label}} for each screenshot.
+        </p>
+      </section>
+    </section>
+    <section class="appendix">
+      <h2>
+        Acknowledgements
+      </h2>
+      <p>
+        TBD
+      </p>
+    </section>
+    <section id="conformance" data-status="done" class="appendix">
```

Comment:
We can probably remove this... this will just be a WG Note, so we don't need a conformance section. We should just make everything descriptive.